ProblemSolvingMentoringSession/Part 2/CheckCountOrExtremum/Solvings/CheckCountOrExtremum_Kharchenko.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+final class CheckCountOrExtremum_Kharchenko: CheckCountOrExtremumProtocol {
```

Comment:
- don't use max() and find extremum by yourself
- use standard .count for finding count in the array